MediaBrowser.Model/Configuration/EncodingOptions.cs

```diff
@@ -275,10 +276,10 @@ public EncodingOptions()
     /// <summary>
     /// Gets or sets the codecs hardware encoding is used for.
     /// </summary>
-    public string[] HardwareDecodingCodecs { get; set; }
+    public IReadOnlyList<string> HardwareDecodingCodecs { get; set; }
```

Comment:
Last I checked `IReadOnlyList` doesn't play well with xml serialization

Reply:
According to this you seem to be correct: https://stackoverflow.com/questions/31552724/how-why-does-xmlserializer-treat-a-class-differently-when-it-implements-ilistt

Will change it back later.

Reply:
AFAIK we tried this multiple times and it always lead to issues.